src/duplicacy_gcdstorage.go

```diff
@@ -27,10 +27,11 @@ import (
 type GCDStorage struct {
 	RateLimitedStorage
 
-	service     *drive.Service
-	idCache     map[string]string
-	idCacheLock *sync.Mutex
-	backoffs    []int
+	service         *drive.Service
+	idCache         map[string]string
+	idCacheLock     *sync.Mutex
+	backoffs        []float64
+	backoffsRetries []int
```

Comment:
Because the variable name could be interpreted ambiguously, I recommend you add a comment to say that this is the number of backoffs that have been performed so far for the current send attempt on the given thread. Also, it's a small thing, but I think the variable name would be more clear as backoffRetries (or as backoffRetriesPerThread) than as backoffsRetries.